src/Components/Search/SearchBar.tsx

```diff
@@ -78,7 +82,7 @@ export class SearchBar extends Component<Props, State> {
       },
       null,
       error => {
-        if (error) console.error(error)
+        if (error) logger.error(error)
```

Comment:
This seems out of scope for this PR - can you talk about what's going on here?

Reply:
Just addressing some feedback from a prior PR- https://github.com/artsy/reaction/pull/2007#discussion_r255221517

This is the better way to do logging- it includes the file location and will also send to Sentry.